docs/reference/how-to/size-your-shards.asciidoc

```diff
@@ -544,3 +544,48 @@ PUT _cluster/settings
   }
 }
 ----
+
+[discrete]
+==== Number of documents in the index shard cannot exceed [2147483519]
+
+
+Elasticsearch shards reflect Lucene's underlying https://github.com/apache/lucene/issues/5176[index 
+`MAX_DOC` hard limit] of 2,147,483,519 (`(2^31)-129`) docs. This figure calulates 
+from the sum of `docs.count` plus `docs.deleted` as reported by <<indices-stats,Index Stats>> 
+per shard. Attempting to surpass this limit, e.g. via ingestion's <<index-modules-translog,translog>>, 
+will cause error 
+
+[source, txt]
+----
+Elasticsearch exception [type=illegal_argument_exception, reason=Number of documents in the shard cannot exceed [2147483519]]
+----
+
+or this error's sister stack trace will appear 
+
+[source, txt]
+----
+Caused by: java.lang.IllegalArgumentException: number of documents in the index cannot exceed 2147483519
+----
+
+TIP: This calculation works off all docs within the shard, including nested docs. 
+Therefore, it's results are not guaranteed to line up to the <<search-count,Count API>> 
```

Comment:
```suggestion
```

Reply:
I agree, but in that case let's drop the preceding line which says the same thing. Also "line up" is too colloquial for my taste, let's use some clearer wording to help folks that might be reading these docs in a foreign language.

Reply:
My [suggestion](https://github.com/elastic/elasticsearch/pull/110449#discussion_r1674232714) maintains this info I think, just all on one line